models/comments-models.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection");
+
+exports.removeComment = (id) => {
+  const commentExists = `SELECT * FROM comments WHERE comment_id = $1`;
+
+  return db
+    .query(commentExists, [id])
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject({ status: 404, message: "Comment not found." });
+      }
+      return result.rows;
+    })
+    .then(() => {
+      const queryString = `DELETE FROM comments WHERE comment_id = $1`;
+      return db.query(queryString, [id]);
+    });
+};
```

Comment:
Here, you return result.rows but then don't use it. It would be more efficient to put all this in the above `.then()` block